src/main/scala/org/broadinstitute/dsde/vault/common/util/ServerInitializer.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.vault.common.util
+
+import akka.actor.Status.{Failure}
+import akka.actor.{ActorSystem, Props}
+import akka.io.IO
+import akka.io.Tcp.CommandFailed
+import akka.pattern.ask
+import akka.util.Timeout
+import spray.can.Http
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+object ServerInitializer {
+
+  def startWebServiceActors(props: Props, webserviceInterface: String, webservicePort: Int, timeoutLong: Long, actorSystem: ActorSystem): Unit = {
```

Comment:
Overall looks good. Minor preferences:

- Don't be like me. Add scaladoc.
- Rename the `timeoutLong` to `timeoutSeconds: Long` (better name for consumers), or better yet, maybe take the value in as a `timeoutDuration: scala.concurrent.duration.Duration`.